test/test_google_cloud_manage.py

```diff
@@ -1072,5 +1079,78 @@ def test_service_account_keys_when_empty(test_cloud_manager):
     )
 
 
+def test_get_project_members_with_wrong_project_id(test_cloud_manager):
+    """
+    """
+    faked_reponse_body = {
+        "error": {
+            "code": 403,
+            "message": "The caller does not have permission",
+            "status": "PERMISSION_DENIED"
+        }
+    }
+
+    test_cloud_manager._authed_session.post.return_value = _fake_response(
+        403, faked_reponse_body)
+
+    with pytest.raises(Exception) as excinfo:
+        test_cloud_manager.get_project_members()
+    assert str(
+        excinfo.value) == 'Unable to get IAM policy for project test_project. The status code is 403'
```

Comment:
I would advice against hardcoding checks against errors strings. If you change it to a custom error type, you should just check if that's the error type that gets raised. e.g. with `pytest.raises(GoogleApiError)`

Reply:
Fixed at 20b0fe69f63d83f478367e6011638d17930e7e62